js/src/dropdown.js

```diff
@@ -39,6 +39,8 @@ const Dropdown = (() => {
   const ARROW_DOWN_KEYCODE       = 40 // KeyboardEvent.which value for down arrow key
   const RIGHT_MOUSE_BUTTON_WHICH = 3 // MouseEvent.which value for the right button (assuming a right-handed mouse)
   const REGEXP_KEYDOWN           = new RegExp(`${ARROW_UP_KEYCODE}|${ARROW_DOWN_KEYCODE}|${ESCAPE_KEYCODE}`)
+  const POPPER_OFFSET_KEY        = 'offset'
+  const POPPER_OFFSET_FN_KEY     = 'fn'
```

Comment:
Not a big fan to add new const for that :thinking:

Reply:
Agree, const is redundant. Updated